### PR TITLE
Api validations

### PR DESCRIPTION
**Pull request by jhan-dev**

### CODE REVIEW

**Review by triciasykes — CHANGES_REQUESTED** [review]
Check your status codes in the update tests.  I don't think you should have if statements

File: spec/requests/memes_spec.rb [hunk 1]
```diff
@@ -2,6 +2,157 @@
...
+          description: 'super cool meme'
+        }
+      }
+      post '/memes', params: meme_params
+      p 'STATUS HERE', response.status
+
+      if response.status == 422
+        expect(response.status).to eq(422)
+        json = JSON.parse(response.body)
+        expect(json['description']).to include 'cannot be blank'
```
Comment (triciasykes) [inline spec/requests/memes_spec.rb]: forgot one

**Review by triciasykes — CHANGES_REQUESTED** [review]
see comment below

**Review by triciasykes — APPROVED** [review]

File: spec/requests/memes_spec.rb (hunk 1 above)
Comment (jhan-dev) [reply spec/requests/memes_spec.rb]: Corrected thank you

**Pull request merged**